scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -57,6 +57,21 @@ object URIHasher {
  * }
  * }}}
  *
+ * example with Execution:
+ * {{{
+ * object YourExecJob extends ExecutionApp {
+ *  override def job = Execution.getConfig.flatMap { config =>
+ *    val theCachedFile = DistributedCacheFile("hdfs://ur-namenode/path/to/your/file.txt")
```

Comment:
won't this fail, as there is no implicit mode available for DistributedCacheFile?